### PR TITLE
Add import group option to the test for  /place/stat/date/within-place

### PR DESCRIPTION
**Pull request by beets**

### CODE REVIEW

File: internal/server/place/golden/get_place_stat_date_within_place_test.go [hunk 1]
```diff
@@ -27,57 +27,65 @@ import (
...
+	for _, opt := range []*e2e.TestOption{
+		{UseCache: true, UseMemdb: true},
+		{UseCache: true, UseMemdb: true, UseImportGroup: true},
 	} {
-		resp, err := client.GetPlaceStatDateWithinPlace(ctx, &pb.GetPlaceStatDateWithinPlaceRequest{
-			AncestorPlace: c.ancestorPlace,
-			PlaceType:     c.placeType,
-			StatVars:      c.statVars,
-		})
+		client, _, err := e2e.Setup()
```
Comment (shifucun) [inline internal/server/place/golden/get_place_stat_date_within_place_test.go:36]: need to pass in opt here "e2e.Setup(opt)"

**Review by shifucun — APPROVED** [review]
This is missed! Thanks for catching up and the fix! Please rerun with the "opt" passed in.

File: internal/server/place/golden/get_place_stat_date_within_place_test.go (hunk 1 above)
Comment (beets) [reply internal/server/place/golden/get_place_stat_date_within_place_test.go]: thanks for the catch! fixed. oddly, no change in the results ...

Comment (shifucun) [reply internal/server/place/golden/get_place_stat_date_within_place_test.go]: That's good, so there are no diff between current and IG!

**Pull request merged**